src/main/java/org/jabref/gui/groups/GroupTreeView.java

```diff
@@ -117,7 +117,7 @@ public GroupTreeView(TaskExecutor taskExecutor,
         this.aiService = aiService;
 
         createNodes();
-        this.getStylesheets().add(Objects.requireNonNull(GroupTreeView.class.getResource("GroupTree.css")).toExternalForm());
+        this.getStylesheets().add(Objects.requireNonNull(GroupTreeView.class.getResource("/org/jabref/gui/Base.css")).toExternalForm());
```

Comment:
I don't think this is even necessary, the Base.cs stylesheet is applied globally, so it would be automatically applied to all nodes in the hierarchy. However, you probably need to add custom classes for the group tree etc to prevent side effects with other elements 

see my PR for an example 
https://github.com/JabRef/jabref/pull/11881

Reply:
i think  @harry924 this means, remove this line.